src/camera.py

```diff
@@ -0,0 +1,105 @@
+from os import getcwd
+
+from picar.SunFounder_PCA9685 import Servo
+from picar import filedb
+
+
+'''
+Camera class to control panning and tilting.
+
+Creates camera object that can be operated on using pan() and tilt().
+pan() operates [0,180] degrees. tilt() operates [0,150] degrees. These 
+functions can be calibrated using calibrate_pan() and calibrate_tilt().
+Calibration adds values to the pan() or tilt().
+'''
+class Camera:
+    PAN_CHANNEL = 1
+    TILT_CHANNEL = 2
+    MAX_PAN_DEGREE = 180
+    MAX_TILT_DEGREE = 150
+    MIN_DEGREE = 0
+    OFFSET_TO_SERVO = 0
+
+    def __init__(self, bus_number=1):
```

Comment:
will add bus_number to constants

Reply:
No, because it changes depending on who calls it.

if I create an object like this: `camera = Camera(bus_number=10)`, I changed the value of bus number. Compare that functionality to that of the rest of the constants.